x-pack/plugins/ml/public/application/model_management/force_stop_dialog.tsx

```diff
@@ -40,15 +40,26 @@ export const StopModelDeploymentsConfirmDialog: FC<ForceStopModelConfirmDialogPr
     {}
   );
 
+  const trainedModelDeployments = useMemo<string[]>(() => {
+    return (
+      model.deployment_ids
+        // Filter out deployments that are used by inference services
+        .filter((deploymentId) => {
+          if (!model.inference_apis) return true;
```

Comment:
This should be using `hasInferenceServices`

Reply:
We need the actual **deployment** IDs, not just an indication that the **model** is referenced by inference services.